src/recensio/plone/browser/templates/results-listing.pt

```diff
@@ -34,7 +34,7 @@
                b_size request/b_size | b_size;
                b_start python:0;
                b_start request/b_start | b_start;
-               batch python:Batch(results, b_size, int(b_start), orphan=0);
+               batch python:isinstance(results, Batch) and results or Batch(results, b_size, int(b_start), orphan=0);
```

Comment:
```suggestion
               batch python:results if isinstance(results, Batch) else Batch(results, b_size, int(b_start), orphan=0);
```
Does that work too?
I try to avoid conditional evaluation as a side effect of a boolean expression as this can lead to unintended results - e.g. if `results` evaluates to False but is a instance of `Batch`.

Reply:
Thanks, that was what I meant to go for, but it must have gotten mixed up on the way from my brain to the template...